test/e2e/traceflow_test.go

```diff
@@ -320,3 +331,19 @@ func (data *TestData) createNPAllowAllEgress(name string) (*networkingv1.Network
 	}
 	return data.createNetworkPolicy(name, spec)
 }
+
+// waitForNetworkpolicyRealized waits for the NetworkPolicy to be realized by the antrea-agent Pod.
+func (data *TestData) waitForNetworkpolicyRealized(pod string, networkpolicy string) error {
+	if err := wait.Poll(200*time.Millisecond, 5*time.Second, func() (bool, error) {
+		cmds := []string{"antctl", "get", "networkpolicy", networkpolicy, "-n", testNamespace}
+		if _, _, err := runAntctl(pod, cmds, data); err != nil {
+			return false, err
```

Comment:
when the networkpolicy is not found, the exit code is not 0, does `runAntctl` return error? If it returns, `wait.Poll` will exit with error immediately.

Reply:
Thanks for point out. Yes, it should return false, nil.